spec/user_can_delete_game_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+feature 'only admin or users can delete a game' do
+  let(:game) { FactoryGirl.create(:game) }
+  scenario 'user can delete game' do
+    sign_up
```

Comment:
Can any user delete a game? If only the user could edit/delete a game, then `game.user` is the user you care about.